keyboards/churrosoft/deck8/info.json

```diff
@@ -0,0 +1,48 @@
+{
+  "keyboard_name": "Deck-8",
+  "manufacturer": "Churrosoft",
+  "url": "https://churrosoft.ar/deck",
+  "maintainer": "Polsaker",
+  "usb": {
+    "vid": "0x1BE2",
```

Comment:
You generally shouldnt use an existing vendor ID that is already allocated.

Reply:
Good catch, I'm changing it to 0xCBBC which doesn't seem to be allocated according to [this](https://www.usb.org/sites/default/files/vendor_ids082119_0.pdf)